### PR TITLE
#169542004 start using app versioning

### PR DESCRIPTION
**Pull request by Pacifique78**
#### What does this PR do?
This PR adds logics for app versioning

#### What are the relevant pivotal tracker stories?
#169542004

#### Screenshots (if appropriate)
Screenshot (121)
Screenshot (123)

### CODE REVIEW

File: Server/V1/Controllers/UserControllers.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        return res.status(200).json({
+            status: 200,
+            message: 'User logged in successfully',
+            data: {
+                token,
+            },
+        });
+    }
+}
+const newClass = new UsersClass();
```
Comment (NkFab) [inline Server/V1/Controllers/UserControllers.js]: Why are you exporting an instance of the class?

Comment (Pacifique78) [reply Server/V1/Controllers/UserControllers.js]: I was exporting an instance to be used outside but I can also export a class and create an instance after.

Comment (NkFab) [reply Server/V1/Controllers/UserControllers.js]: So if you export the class itself will it not work?

Comment (Pacifique78) [reply Server/V1/Controllers/UserControllers.js]: I have made the changes and it is working

**Pull request merged**